docs/code-of-conduct-response.rst

```diff
@@ -0,0 +1,58 @@
+
+Write the Docs Code of Conduct - Reporting Guide
+================================================
+
+.. _report: conduct@writethedocs.org
+
+In general, in any Write the Docs space (conferences, meetup, Slack, forum), if you believe someone is violating the code of conduct we ask that you report it to the staff by emailing report_. All reports will be kept confidential. In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate law enforcement first. If you are unsure what law enforcement agency is appropriate, please include this in your report and we will attempt to notify them.
+
+If the violation occurs at an event (conference, meetup), get in touch with one of the event staff right away, in any of the following ways:
+
+* In person
+* By phone
+* By email
+
+In your report please include:
+
+* Your contact information (so we can get in touch with you if we need to follow up)
+* Names of any individuals involved (real names, nicknames, or pseudonyms). If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public chat logger) please include a link.
+* Any extra context you believe existed for the incident.
+* Whether you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+What happens after you file a report
+------------------------------------
+
+You will receive an email from the WTD staff acknowledging receipt. We promise to acknowledge receipt within 24 hours, and will aim for much quicker than that.
+
+The staff will meet immediately to review the incident and determine the following:
+
+* What happened.
+* Whether this event constitutes a code of conduct violation.
+* Who the bad actor was.
+* Whether this is an ongoing situation, or if there is a threat to anyone's physical safety.
+* If this is determined to be an ongoing incident or a threat to physical safety, the staff's immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+After the staff has a complete account of the events they will make a decision as to how to response. Responses may include:
+
+* Nothing (if we determine no violation occurred).
+* A private reprimand from the staff to the individual(s) involved.
+* A public reprimand.
+* An imposed vacation (for example, asking someone to "take a week off" from Slack or the forum).
+* A permanent or temporary ban from some or all WTD spaces (Slack, the forum, meetups).
+* A request for a public or private apology.
```

Comment:
Since this seems to be in order of severity, I'd put the "apology" line before "A public reprimand"